docs/modules/rust-guide/pages/rust-quickstart.adoc

```diff
@@ -271,7 +271,7 @@ This definition specifies that the data passed to the `+print+` function is retu
 [[start-the-local-network]]
 == Start the local network
 
-Before you can build your project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a sub-network that you can access.
+Before you can build your project, you need to connect to the {IC} network either running locally in your development environment or the decentralized {IC} mainnet.
```

Comment:
See above.

Reply:
I'll just resolve the duplicates and we can sort the IC term on the first comment.